src/Components/calculate.test.js

```diff
@@ -0,0 +1,5 @@
+import calculate from '../Logic/Calculate';
+test('adds two numbers to return an answer', () => {
+  const object = { total: '2', next: '3', operation: '+' };
+  expect(calculate(object, '=').total).toBe('6');
```

Comment:
- [ ] This test is failing, please checkout the screenshot below and fix the issue.

![failedTest](https://user-images.githubusercontent.com/62849872/171429311-5362d361-2b11-4890-ba83-a30a8576a9bf.png)